rancher-metadata-syncer/deployment-proxy.yaml

```diff
@@ -0,0 +1,61 @@
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    app: rancher-metadata
+  name: rancher-metadata
+  namespace: cattle-system
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: rancher-metadata
+  template:
+    metadata:
+      labels:
+        app: rancher-metadata
+    spec:
+      containers:
+      - env:
+        - name: HTTPS_PROXY
+          value: https://<user>:<password>@<ip_addr>:<port>/
+        - name: HTTP_PROXY
+          value: http://<user>:<password>@<ip_addr>:<port>/
+        image: cube8021/rancher-metadata-airgap:latest
```

Comment:
Based on the feedback from @jambajaar , we need to put the images to `rancher/support-tools`

Reply:
I have submitted a request for a docker hub repo at rancher/metadata-syncer to replace this image.